protocol/daemons/liquidation/client/client.go

```diff
@@ -79,9 +86,11 @@ func (c *Client) Start(
 		}
 	}()
 
-	subaccountQueryClient := satypes.NewQueryClient(queryConn)
-	clobQueryClient := clobtypes.NewQueryClient(queryConn)
-	liquidationServiceClient := api.NewLiquidationServiceClient(daemonConn)
+	// Initialize the query clients. These are used to query the Cosmos gRPC query services.
+	c.BlocktimeQueryClient = blocktimetypes.NewQueryClient(queryConn)
+	c.SubaccountQueryClient = satypes.NewQueryClient(queryConn)
+	c.ClobQueryClient = clobtypes.NewQueryClient(queryConn)
+	c.LiquidationServiceClient = api.NewLiquidationServiceClient(daemonConn)
```

Comment:
let me know if anyone wants me to save the queryConn instead of the clients explicitly. 

probably not a big deal since these will be removed in `Liquidation Daemon Improvement (Part 2)`